Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -1346,6 +1347,26 @@ void webkit_web_context_set_sandbox_enabled(WebKitWebContext* context, gboolean
     context->priv->processPool->setSandboxEnabled(enabled);
 }
 
+static bool pathIsHomeDirectory(const char* path)
+{
+    GUniquePtr<char> resolvedPath(realpath(path, nullptr));
+    if (!resolvedPath) {
+        g_warning("Failed to canonicalize path %s: %s", path, g_strerror(errno));
+        return true;
+    }
+
+    if (!strcmp(resolvedPath.get(), "/home"))
+        return true;
+
+    GUniquePtr<char> resolvedHomeDirectory(realpath(g_get_home_dir(), nullptr));
+    if (!resolvedPath) {
+        g_warning("Failed to canonicalize path %s: %s", g_get_home_dir(), g_strerror(errno));
+        return true;
+    }
+
+    return !strcmp(resolvedPath.get(), resolvedHomeDirectory.get());
```

Comment:
I was going to comment this being case-sensitive is sometimes wrong. But it turns out GLib assumes this everywhere already so its just not supported.

Note that bubblewrap does go out if its way to handle case-insensitive filesystems.

Reply:
Updated.